.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - name: Set Swap Space
-        uses: pierotofy/set-swap-space@49819abfb41bd9b44fb781159c033dba90353a7c
+        uses: pierotofy/set-swap-space@0404882bc4666c0ff2f6fd8b3d32af69a730183c
```

Comment:
can we have the version in the comment like in dependabot?

Reply:
For actions where we are pinning to the version we can, here we are pinning to the commit digest directly as they are not releasing new versions from 2021